tools/perspective-scripts/sh.mjs

```diff
@@ -323,7 +323,7 @@ function parse_bind_args(fragments, ...args) {
     for (let i = 0; i < fragments.length - 1; i++) {
         const arg = args[i];
         const start = terms.length === 0 ? fragments[i] : terms.pop();
-        if (arg === undefined || arg !== arg || arg === false) {
+        if (arg === undefined || Number.isNaN(arg) || arg === false) {
```

Comment:
Unrelated change, I was poking around in here and was a little bit surprised by seeing `x !== x`. I think `Number.isNan()` does the same thing, although to be fair I haven't seen the Wat talk in a minute.